src/main/java/seedu/duke/commands/MarkCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.Item;
+import seedu.duke.data.ShoppingList;
+
+public class MarkCommand extends Command {
+    public static final String COMMAND_WORD = "MARK";
+    public static final String SUCCESS_DONE = "Yes! I've marked this item as bought: "
```

Comment:
You could rename to MESSAGE_SUCCESS/SUCCESS_MESSAGE :) apologies for the caps haha I'm not screaming... (it's just to match the coding standard of a final variable ) 😰

Reply:
Thanks!! Will change it!!